docs/developers/guides/incentives_registration.md

```diff
@@ -0,0 +1,163 @@
+
+### The Voting Period
+
+At this point you'll want to track which validator has voted and which has not. You'll want to re-engage directly with top stake-holders, ie. the highest-ranking validator operators, to ensure that:
+
+1. they are aware of your proposal;
+2. they can ask you any questions about your proposal; and
+3. they are prepared to vote.
+
+Remember that any voter may change their vote at any time before the voting period ends. That historically doesn't happen often, but there may be an opportunity to convince a voter to change their vote. The biggest risk is that stakeholders won't vote at all (for a number of reasons). Validator operators tend to need multiple reminders to vote. How you choose to contact validator operators, how often, and what you say is up to you--remember that no validator is obligated to vote, and that operators are likely occupied by competing demands for their attention. Take care not to stress any potential relationship with validator operators.
+
+## Incentives Analysis through Telemetry
+
+### Telemetry Basics & Setup
+
+The telemetry package of the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk) allows operators and developers to gain insight into the performance and behavior of their applications.
+
+To enable telemetrics, set `telemetry.enabled = true` in the `app.toml` config file of the node. The Cosmos SDK currently supports enabling in-memory and [Prometheus](https://prometheus.io/) telemetry sinks. The in-memory sink is always attached (when telemetry is enabled) with a ten second interval and one minute retention. This means that metrics will be aggregated over ten seconds, and metrics will be kept alive for one minute. To query active metrics, set `api.enabled = true` in the `app.toml`. This exposes a single API endpoint: `http://localhost:1317/metrics?format={text|prometheus}`, the default being `text`.
+
+### Emitting & Collecting Metrics
+
+If telemetry is enabled via configuration, a single global metrics collector is exposed via the [`go-metrics`](https://github.com/armon/go-metrics) library. This allows for emitting and collecting metrics through a simple [API](https://github.com/cosmos/cosmos-sdk/blob/v0.46.0-rc1/telemetry/wrapper.go). For example:
+
+```go
+func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
+  defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyEndBlocker)
+
+  // ...
+}
+```
+
+Developers may use the [`telemetry`](https://docs.evmos.org/protocol/telemetry.html) package directly, which provides wrappers around metric APIs that include adding useful labels, or they must use the `go-metrics` library directly. It is preferable to add as much context and adequate dimensionality to metrics as possible, so the `telemetry` package is advised.
+
+Regardless of the package or method used, the Cosmos SDK supports `guages`, `summaries`, and `counters` as metric types.
```

Comment:
this is not relevant for a developer since they won't modify the golang code, they just need to know how to use the telemetry